data/migrations/20190910143517_createDB.js

```diff
@@ -0,0 +1,36 @@
+
+exports.up = function(knex) {
+    return knex.schema.createTable("cars", table => {
+        table.increments();
+
+        table
+            .string("vinNumber",17)
+            .unique()
+            .notNullable()
+
+        table
+            .string("make", 128)
+            .notNullable()
+
+        table
+            .string("model",128)
+            .notNullable()
+        
+        table
+            .integer("mileage")
+            .notNullable()
+
+        table
+            .string("transmissionType", 128)
+        
+        table
+            .string(titleStatus, 128)
```

Comment:
```suggestion
            .string("titleStatus", 128)
```
This needs to be wrapped in quotation marks. I wasn't able to migrate:latest and seed:run until I did this.